NebulaModel/Packets/GameHistory/GameHistoryRemoveTechPacket.cs

```diff
@@ -2,13 +2,13 @@
 {
     public class GameHistoryRemoveTechPacket
     {
-        public int Index { get; set; }
+        public int techId { get; set; }
```

Comment:
There is naming convention to have first letter capital for the class properties :)